src/services/EdFi.DataManagementService.Api.Tests.E2E/Management/ContainerSetup.cs

```diff
@@ -17,7 +17,6 @@ public async Task<string> SetupDataManagement()
         var dockerImage = new ContainerBuilder()
             .WithImage(imageName)
             .WithPortBinding(8080)
-            .WithWaitStrategy(Wait.ForUnixContainer().UntilHttpRequestIsSucceeded(r => r.ForPort(8080)))
```

Comment:
When we have invalid appsettings, this WaitStrategy will cause the E2E test suite to run indefinitely, because it is waiting for a successful Http response, which it is never going to get. Removing this line will simply fail the tests in that scenario.